nginx/nginx.conf

```diff
@@ -25,6 +25,9 @@ http {
     location /static {
       alias /usr/share/nginx/html/static;
     }
+    location /media {
+        alias /usr/share/nginx/html/media;
+    }
```

Comment:
This nginx.conf is used in many setups. So I don't think it is a good idea to add it here and only provide a solution for docker-compose.

Reply:
I couldn't find any other place it is being used

Reply:
@alles-klar are you thinking of kubernetes? i don't have a strong opinion on this, but if the images are already not working, this is a start... :)

Reply:
Sorry for the long delay. I think all setups are using the nginx config - also the productive ones. If we add the /media path there, all data that is stored there is free accessible without any authentication. If user add confidential pictures this is a security issue.

Reply:
the production docker-compose setup is a container purely meant for defect dojo. if somebody would  put sensitive data in there I don't think they know what they are doing :-). Also the production (release) config doesn't map any local files to the nginx media folders.  `setup.bash` doesn't use this `nginx.conf`.

Reply:
Using this setup just for debugging and development is fine but I am afraid that someone adopt this solution for other setups. Maybe we add a comment in the nginx config that it is currently for development only with docker-compose only and that this directory has to be empty in production?